volatility3/framework/plugins/linux/lsblk.py

```diff
@@ -0,0 +1,447 @@
+from typing import List
```

Comment:
Please add the standard header (and ensure you're happy with the copyright) so that we can accept this.